### PR TITLE
First code review

### PR DESCRIPTION
**Pull request by Artbsc1992**
Reviewing previous code from trials.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<body>
```
Comment (Artbsc1992) [inline index.html:1]: - I kindly recommend that you used HTML 5 semantic header at the start of your HTML file.
- Please add the <link> tag so you can link your style sheet.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    align-items: center;
+  }
+  
+  #survey-form{
+    background-color: rgba(137, 43, 226, 0.486);
+    width: 40%;
+    padding: 20px;
+    border-radius: 5px;
+    margin-top: 50px;
+  }
```
Comment (Artbsc1992) [inline style.css]: Everything looks to work, you only need to review your semantic errors and you will be ready to go. :smiley:

**Review by Artbsc1992 — COMMENTED** [review]
image

# Changes required :recycle: 
You did a good job creating your first project, please read my comments for guidance.

Submit for a re-review when you are finished.

Thank you :smile_cat:

File: style.css (hunk 2 above)
Comment (OLIPLICHE) [reply style.css]: ## OPTIONAL
- [x] I suggest that you leave the comment under a line or block of code that fits the context the comment("semantic errors") is about. 

- [x] Please leave the comment under the HTML block of code if you want to make a general statement.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Artbsc1992, 

Good job so far!
There are some issues that you still need to work on to get approved but you are almost there!


## Required Changes ♻️
- Kindly Fix the Linter 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      </div>
+  
+      <div>
+        <textarea name="text" id="" cols="50" rows="10" placeholder="Enter your comment here..."></textarea>
+      </div>
+  
+      <button id="submit">submit</button>
+    </form>
+  </body>
+  </html>
```
Comment (Artbsc1992) [inline index.html]: - I kindly recommend that you used HTML 5 semantic header at the start of your HTML file.
- Please add the "link" tag so you can link your style sheet.

**Comment by Artbsc1992** [comment]
Hi @OLIPLICHE, thank you for your feedback, please can you be more clear about how I should fix the linter because I thought linters had to show error, that is why I'm reviewing it, right?
Or maybe I shouldn't put linterns inside the repo?.

**Comment by OLIPLICHE** [comment]
> I understand you perfectly. Now you can just Just fix the Linter for the PR to be approved since you are going to merge it. 👍

**Review by zainsadaqat — APPROVED** [review]
# Hello @Artbsc1992 😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

[Optional]

- [x] Please ask the student to create a descriptive Pull Request. For example, what he has done in this pull request. 👌🏻 
- [x] Please ask the student to use the correct GitHub Flow. As you can see, master and main both work the same and it may cause some conflicts that's it's better to name branch with some feature name. 👍🏻 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**